include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -114,7 +114,7 @@ SECTIONS
 
     _image_rom_start = ROM_ADDR;
 
-    SECTION_PROLOGUE(_TEXT_SECTION_NAME,,)
+    SECTION_PROLOGUE(rom_start,,)
```

Comment:
ideally this would be defined in the same header that defines _TEXT_SECTION_NAME

Reply:
For now, I am trying to align with what other archs are doing; but sure, `_ROM_START_SECTION_NAME` or something like that would have been more appropriate.

I will look into fixing that name for all archs in a separate PR.